README.md

```diff
@@ -39,18 +39,30 @@ built from source using the latest version of [Go](https://golang.org), and the
 available instructions in our
 [Wiki](https://github.com/git-lfs/git-lfs/wiki/Installation#source).
 
-### Usage
+#### Installing from a [binary package](https://github.com/git-lfs/git-lfs/releases)
```

Comment:
I have a few suggestions on this line:

1. What do you think about breaking this into an H3-level "Installing" heading and two H4-level sub-headings "From binary" and "From source"? 
2. Typically we don't have links in headings, so maybe you could write a short 1-sentence below the heading and mention this link.